homework/PavelBo/collections/wordCounter/src/test/java/com/tutrit/java/quickstart/UniqueWordCounterTest.java

```diff
@@ -0,0 +1,44 @@
+package com.tutrit.java.quickstart;
+
+import junit.framework.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.Assert.assertEquals;
+
+public class UniqueWordCounterTest {
+
+    UniqueWordCounter uniqueWordCounter;
+
+    @Before
+    public void setup(){
+        uniqueWordCounter = new UniqueWordCounter();
+        addInitialWords();
+    }
+
+    @Test
+    public void addWord_test(){
+        addInitialWords();
```

Comment:
You use it method in method: @Before setup

Reply:
therefore second call in test method is redundant.